saleor/dashboard/order/views.py

```diff
@@ -92,6 +94,11 @@ def order_add_note(request, order_pk):
             'Added note')
         order.create_history_entry(comment=msg, user=request.user)
         messages.success(request, msg)
+        if note.is_public:
+            order_url = build_absolute_uri(
+                reverse('order:details', kwargs={'token': order.token}))
+            email = order.get_user_current_email()
+            send_note_confirmation.delay(email, order_url)
```

Comment:
It looks like something the form should take care of?

Reply:
This could be a function in the form e.g. `form.send_note_confirmation(email, order_url)`.